### DISCUSSION TITLE
HD-38 Soil sensor

### DISCUSSION THREAD

**Discussion by toppe** [arendst/Tasmota#11238] [Q&A] [answered]
Hi,

Does some one knows if the HD-38 can work inside Tasmota? It’s this sensor: http://qqtrading.com.my/soil-moisture-sensor-module-hd-38

Any other recommendations for a solid (wired!) soil moisture sensor is welcome any time!

thanks!

**Comment by sfromis** [comment, marked as answer]
On 3.3v, I'd expect the analog output to work the analog input on a dev board (like NodeMCU), which is supported by Tasmota. Assuming that the digital output is switch-like (under/over a certain level), that should work configured as a button or switch input.
